projects/graph/src/graph.py

```diff
@@ -1,9 +1,85 @@
 """
 Simple graph implementation compatible with BokehGraph class.
 """
-
-
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.vertices = {}
+    
+    def add_vertex(self, vertex_id):
+        self.vertices[vertex_id] = Vertex(vertex_id)
+
+    def add_edge(self, v1, v2):
+        if v1 in self.vertices and v2 in self.vertices:
+            self.vertices[v1].edges.add(v2)
+            self.vertices[v2].edges.add(v1)
+        else:
+            raise IndexError("That vertex does not exist!")
```

Comment:
We discussed this in DM. Nice work on using Vertex objects. You will find that will be useful when you need to have a place to store `visited` status, x/y coords, color, etc of given vertex somewhere.